config/armada/config.yaml

```diff
@@ -17,60 +17,99 @@ grpc:
     permitWithoutStream: false
 redis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 0
   poolSize: 1000
 eventsRedis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 0
   poolSize: 1000
 eventsApiRedis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 1
   poolSize: 1000
 defaultToLegacyEvents: true
 scheduling:
   preemption:
-    # if enabled, Armada will accept submitted jobs which have a priorityClassName field
-    # otherwise it will refuse the jobs on submit
-    enabled: false
-    # priorityClasses field is a map of priority class names to priority values
-    # defined priority classes should exist with the same priority in all executor clusters
-    priorityClasses: {}
-    # if preemption is enabled, Armada server will set the priorityClassName in the job's podSpec
-    # with the value of defaultPriorityClass, if the podSpec does not already reference a priority class
+    enabled: true
+    priorityClasses:
```

Comment:
assuming that this file remains the default config we'll need to ensure we create these in the helm chart

Reply:
I added the PCs to the executor helm. Could you please verify that it's correct?